tiled/_tests/test_writing.py

```diff
@@ -174,6 +174,32 @@ def test_write_dataframe_partitioned(tree):
         assert result.specs == specs
 
 
+# @pytest.mark.filterwarnings(f"ignore:{WARNING_PANDAS_BLOCKS}:DeprecationWarning")
```

Comment:
This should probably be deleted. Was this here to silence an unrelated DeprecationWarning that we should address in a separate PR?

Reply:
I agree. This can be deleted.

This is a remnant of some changes I made a while ago. The DeprecationWarning ended up getting handled in the (deprecated) object cache.  This alternate approach seems to have been accidentally left asa comment.

https://github.com/bluesky/tiled/commit/5aae35a7d3b6a3959263b1df656aeb794bd688c5

Reply:
I take it we should remove all 4 of these commented items in this file? Want that in a separate PR or here?

Reply:
Either works.

Reply:
I made a new Issue.

Reply:
https://github.com/bluesky/tiled/issues/772